packages/types/src/base.ts

```diff
@@ -31,6 +31,12 @@ export enum NetworkTypes {
   ETH_RINKEBY = 'ETH_RINKEBY'
 }
 
+export enum UtxoAccountType {
+  SegwitNative = 'SegwitNative',
+  SegwitP2sh = 'SegwitP2sh',
+  P2pkh = 'P2pkh'
+}
```

Comment:
I'm still fundamentally unhappy about the existence (and capitalization!) of this enum, but I can't think of anything better and l'll get over it.